src/pages/scheduler/components/share/ShareLinkOptions.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from 'react';
+
+import { CopyIcon } from '@chakra-ui/icons';
+import { Button, Heading, HStack, Icon, Input, useClipboard, useToast } from '@chakra-ui/react';
+import { HiLink } from 'react-icons/hi';
+import {
+  EmailShareButton,
+  EmailIcon,
+  FacebookShareButton,
+  FacebookIcon,
+  TelegramShareButton,
+  TelegramIcon,
+  WhatsappShareButton,
+  WhatsappIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+
+type SocialMediaButtons_Props = {
+  slug: string;
+};
+
+const SocialMediaButtons = ({ slug }: SocialMediaButtons_Props) => {
+  return (
+    <HStack justify="center">
+      <EmailShareButton children={<EmailIcon size={50} round={true} />} url={slug} />
```

Comment:
```suggestion
      <EmailShareButton children={<EmailIcon size={50} round />} url={slug} />
```
small nit, not even sure if this will work